molgenis-data-annotators/src/test/resources/gonl/test-out-expected.vcf

```diff
@@ -2,4 +2,4 @@
 ##INFO=<ID=GoNL_GTC,Number=.,Type=String,Description="GenoType Counts. For each ALT allele in the same order as listed = 0/0,0/1,1/1,0/2,1/2,2/2,0/3,1/3,2/3,3/3,etc. Phasing is ignored; hence 1/0, 0|1 and 1|0 are all counted as 0/1. When one or more alleles is not called for a genotype in a specific sample (./., ./0, ./1, ./2, etc.), that sample's genotype is completely discarded for calculating GTC.">
 ##INFO=<ID=GoNL_AF,Number=.,Type=String,Description="The allele frequency for variants seen in the population used for the GoNL project">
 #CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO
-1	249239510	test	C	G	.	.	GoNL_GTC=488,10,0;GoNL_AF=0.010040160642570281;
+1	249239510	test	C	G	.	.	GoNL_GTC=488|10|0;GoNL_AF=0.010040160642570281;
```

Comment:
this was a seperate change that's merged already right? not sure why it still shows as a difference.